README.md

```diff
@@ -3,11 +3,9 @@
 ![GitHub top language](https://img.shields.io/github/languages/top/s-suresh-kumar/yard-pal)
 [![GitHub license](https://img.shields.io/github/license/s-suresh-kumar/yard-pal)](LICENSE)
 
-`short description goes here`
+- Yard Pal is an online web application that brings your local yard sales to your computer or device. With this application, you can view your local yard sales or host your own yard sale safely from your home. This way the sellers and buyers can safely communicate with each other and follow social-distancing guidelines. 
 
-`replace this picture with an actual screenshot`
-
-[![screenshot placeholder](https://placekitten.com/500/500)](https://yard-pal.herokuapp.com/)
+![](client/src/images/YPDeployScreen.png)(https://yard-pal.herokuapp.com/)
```

Comment:
Changing to the suggestion below will allow the picture to serve as the link for the deployed app. As is, the url to the app is visible under the picture inside parentheses.

```suggestion
[![screenshot](./client/src/images/YPDeployScreen.png)](https://yard-pal.herokuapp.com/)
```